test/merge/test_file_merge.py

```diff
@@ -0,0 +1,255 @@
+from copy import copy
+from io import StringIO
+from pathlib import Path
+from unittest import TestCase
+
+from clang.cindex import CursorKind
+
+from merge.choice import Choice
+from merge.conflict import Conflict2Way, Conflict3Way
+from merge.file_bit import FileBit
+from merge.file_merge import FileMerge
+import merge.cursor_utils
+
+
+class TestFileMerge(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        from merge.external_parser_setup import ExternalParserSetup
+        ExternalParserSetup.setup()
+
+    def setUp(self):
+        self.fb1 = FileBit(1, ("int main() {\n"
```

Comment:
I still don't understand. 

We have merge examples on Confluence, but they do not relate to code.

Also we have a test project (./testproject) and a couple of sample files (./samples/*) for testing the whole program together. They are where they were, and I run the code on them too.

On the other hand, the tests in this file are "unit-test", since they test only one unit - the merge algorithm. They do not involve the CUI and other helper facilities.